get-involved.php

```diff
@@ -56,14 +56,15 @@
             <h2>General Funding</h2>
             <p>Patreon works like an ongoing crowdfunding campaign. Choose an amount, get rewards, and help us reach our goals. <a class="read-more" href="https://www.patreon.com/" target="blank">Learn More</a></p>
             <p>PayPal is a quick and easy solution. Choose a one-time amount or set up a subscription.</p>
-
+            <p>Buying merch from our store is another way to help us financially, plus you can get some exclusive elementary gear to show your support to friends, family, and coworkers.</p>
             <div class="actions">
                 <a class="button flat suggested-action" href="https://www.patreon.com/elementary" target="_blank">Patreon</a>
                 <form action="https://www.paypal.com/cgi-bin/webscr" id="paypalform" method="post" target="_top">
                 <input type="hidden" name="cmd" value="_s-xclick">
                 <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                 <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                 <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
+                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
                 </form>
```

Comment:
Okay, so the problem is that the form inputs aren't indented so it isn't obvious, but this form tag needs to be moved up a line.

```
                <form action="https://www.paypal.com/cgi-bin/webscr" id="paypalform" method="post" target="_top">
                <input type="hidden" name="cmd" value="_s-xclick">
                <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
                </form>
```

```
                <form action="https://www.paypal.com/cgi-bin/webscr" id="paypalform" method="post" target="_top">
                    <input type="hidden" name="cmd" value="_s-xclick">
                    <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                    <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                    <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
                </form>
                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
```